packages/@aws-cdk/aws-codepipeline-actions/test/s3/s3-deploy-action.test.ts

```diff
@@ -175,6 +176,26 @@ describe('S3 Deploy Action', () => {
       ],
     });
   });
+
+  test('KMSEncryptionKeyARN value', () => {
+    const stack = new Stack();
+    minimalPipeline(stack);
+
+    Template.fromStack(stack).hasResourceProperties('AWS::CodePipeline::Pipeline', {
+      'Stages': [
+        {},
+        {
+          'Actions': [
+            {
+              'Configuration': {
+                'KMSEncryptionKeyARN': Match.anyValue(),
```

Comment:
I have added a value instead of Match.value(). The only thing is though that I do not know what the generated ARN would look like.

Reply:
Hi again @TheRealAmazonKendra .. looks like there was a major refactor on aws-cdk repo .. I have merged those changes and re-added my code. Please let me know if we need to change anything else.. Please advise on Match.anyValue() change as we are dealing with a generated value

Reply:
My initial thought on this was that we need to know the presence of this key (KMSEncryptionKeyARN) in the test .. value is not really our purpose here

Reply:
@hussainak in the generated CloudFormation template, this will be something like 

```json
{ "Fn::GetAtt": ["<key ID>", "Arn"] }
```

 in which `<key ID>` is the logical ID of the KMS key declared elsewhere in the template. This is what you should be checking for.